src/js/components/Form/Form.js

```diff
@@ -165,29 +174,27 @@ const Form = forwardRef(
     ref,
   ) => {
     const { format } = useContext(MessageContext);
-
     const [valueState, setValueState] = useState(valueProp || defaultValue);
     const value = useMemo(
       () => valueProp || valueState,
       [valueProp, valueState],
     );
     const [touched, setTouched] = useState(defaultTouched);
-    const [validationResults, setValidationResults] = useState(
-      defaultValidationResults,
-    );
-    // when onBlur input validation is triggered, we need to complete any
-    // potential click events before running the onBlur validation.
-    // otherwise, click events like reset, etc. may not be registered.
-    // for a detailed scenario/discussion,
-    // see: https://github.com/grommet/grommet/issues/4863
-    // the value of pendingValidation is the name of the FormField
-    // awaiting validation.
-    const [pendingValidation, setPendingValidation] = useState(undefined);
-
+    const [validationResults, setValidationResults] = useState({
+      errors: errorsProp,
```

Comment:
Did we delete `defaultValidationResults`? or is it still being used somewhere else. I didn't see it deleted but seems like we're not using it

Reply:
Being used by `onReset`, L526-ish.